src/index.js

```diff
@@ -53,26 +53,37 @@ window.wikiaJWPlayer = function (elementId, options, callback) {
 		var playerInstance = jwplayer(elementId),
 			videoId = options.videoDetails.playlist[0].mediaid,
 			willAutoplay = options.autoplay,
+			lang = options.lang || 'en',
+			// IMA supports two-letter ISO 639-1 code
+			langForAds = lang.substr(0, 2),
+			i18n = wikiaJWPlayeri18n[lang],
```

Comment:
i'd rename `i18n` to `langObj`

Reply:
this name is consistent with mobile wiki/mercury i18n helper name :) so I'd leave it as is